index.html

```diff
@@ -0,0 +1,736 @@
+                            <div class="f1-flex"> 
+                                <span><a class="f1-by" href="#">Reachel Kaser</a>-</span>
+                                <span>32 minutes ago</span>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="f1">
+                        <a href="#"><img class="f1-img" alt="gallery-1" src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2018/07/weed_tech-796x417.jpg"></a>
+                        <div class="f1-media">
+                            <h4><a class="gallery-link" href="#">Scientist modified a strain of</a></h4>
+                            <h4><a class="gallery-link" href="#">cannabis to kill cancer cells</a></h4>
+                            <div class="f1-flex"> 
+                                <span><a class="f1-by" href="#">Tristan Greene</a>-</span>
+                                <span>18 minutes ago</span>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="f1">
+                         <a href="#"><img class="f1-img" alt="gallery-1" src="https://img-cdn.tnwcdn.com/image?fit=1280%2C720&url=https%3A%2F%2Fcdn0.tnwcdn.com%2Fwp-content%2Fblogs.dir%2F1%2Ffiles%2F2020%2F07%2FUntitled-design-2020-07-28T145238.994.png&signature=7510a610d50d85abc561a0361273e805"></a>
+                        <div class="f1-media">
+                            <h4><a class="gallery-link" href="#">TikTok owner ByteDance's AI chief</a></h4>
+                            <h4><a class="gallery-link" href="#">to leave as US pressure rises</a></h4>
+                            <div class="f1-flex"> 
+                                <span><a class="f1-by" href="#">Thomas Macualay</a>-</span>
+                                <span>7 hours ago</span>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="f1">
+                         <a href="#"><img class="f1-img" alt="gallery-1" src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/CES-2021-796x417.jpg"></a>
+                        <div class="f1-media">
+                            <h4><a class="gallery-link" href="#">CES officially goes fully digital in</a></h4>
+                            <h4><a class="gallery-link" href="#">2021</a></h4>
+                            <div class="f1-flex"> 
+                                <span><a class="f1-by" href="#">Rachel Kaser</a>-</span>
+                                <span>5 hours ago</span>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="flex">
+                    <h2 ><a class="link" href="#">Latest funding rounds on Index.co ></a></h2>
+                    <div class="flex">
+                        <a class="link-2" href="#">Add info</a>
```

Comment:
- [ ] For **desktop** notice that there are 4 elements in the original
![image](https://user-images.githubusercontent.com/53256139/89110487-acaff780-d410-11ea-9305-f97d77429a17.png)